js/btcex.js

```diff
@@ -1284,7 +1287,7 @@ module.exports = class btcex extends Exchange {
             'instrument_name': market['id'],
             'type': type,
         };
-        if (side === 'sell') {
+        if (side === 'sell' || market['spot']) {
```

Comment:
Market buys require the cost/quote amount even for spot markets and that logic is added below. Why do we need to add it here?

Reply:
oh, maybe I made mistake but now it impossible to create `buy` `limit` spot order, because we don't specify amount for this types of order.
`BadRequest btcex {"id":"1678269469","jsonrpc":"2.0","usIn":1678269469646,"usOut":1678269469661,"usDiff":15,"error":{"code":8000,"message":"Request params not valid!","data":{}}}` because we don't send amount
What is the best way to fix this bug?

Reply:
@ndubel I will test it, we can probably replace `market['spot']` with `type === 'limit'`